client/reader/following/intro.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import { hasGraduatedRecommendations } from 'state/reader/start/selectors';
+import { requestGraduate } from 'state/reader/start/actions';
+
+const FollowingIntro = ( props ) => {
+	if ( hasGraduatedRecommendations || hasGraduatedRecommendations === null ) {
+		return null;
+	}
+
+	const handleIntroClose = () => {
```

Comment:
as a general practice for functional components its better to have helper functions exist outside of the component that way it gets reused.

e.g.

```jsx
const handleIntroClose = props => props.requestGraduate();
const FollowingIntro = ( props ) => {
 ...
}
```

in this case it doesn't matter because we will only ever instantiate one.